src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -200,6 +202,7 @@ public void should_keep_zip_file_after_unsuccessful_upload_and_not_create_doc_pr
         CloudBlockBlob blob = testContainer.getBlockBlobReference(zipFile);
         await("file should not be deleted")
             .timeout(2, SECONDS)
+            .pollInterval(1, TimeUnit.SECONDS)
```

Comment:
Here we need to wait the full time (the background job may have not even checked the file)

Reply:
In fact it waits until the timeout expires unless the file is deleted before that. This way it just the condition more often...

Reply:
To clarify, here we check that something *didn't happen*, so we can't check immediately.
Other cases are fine.